src/ol/layer/Base.js

```diff
@@ -288,11 +289,11 @@ class BaseLayer extends BaseObject {
   }
 
   /**
-   * Sets the backgrlound color.
+   * Sets the background color.
    * @param {BackgroundColor} [opt_background] Background color.
    */
   setBackground(opt_background) {
-    this.background_ = opt_background;
+    this.background_ = opt_background !== undefined ? opt_background : false;
```

Comment:
It might make sense to change the zero value to an empty string (instead of `false`) here.  Either way, it looks like this might impact other code.

Reply:
Changing the default would be a bigger change since the false value seems documented for MapboxLayer: https://openlayers.org/en/latest/apidoc/module-ol_layer_MapboxVector-MapboxVectorLayer.html. 
Maybe this change should be reverted and tackled as a whole in another PR (and also looking at all `getBackground` and `BackgroundColor` usages. Possibly adding `undefined` to BackgroundColor). What do you think about this?